protocol/communities/manager.go

```diff
@@ -1707,10 +1701,6 @@ func (m *Manager) HandleCommunityRequestToJoin(signer *ecdsa.PublicKey, request
 			if !matching {
 				// if ownership of only one wallet address cannot be verified,
 				// we mark the request as cancelled and stop
-				err = m.markRequestToJoinAsCanceled(signer, community)
-				if err != nil {
-					return nil, err
-				}
```

Comment:
This looked suspicious first.. I wonder why we marked the request to join as `canceled` here.
Should've probably rather been marked as `declined` in the first place.

Reply:
It was redundant, it will be indirectly invoked further in the processing pipeline, here:
https://github.com/status-im/status-go/blob/5543a29f6250d922eee70571e2b1590209b798d5/protocol/messenger_handler.go#L1425-L1434

Reply:
That's correct, I still wonder why it was "mark as cancelled" in the first place

Reply:
I presume that the redundancy was unintentionally introduced. The tests are passing, so I'll take that as an indication that there's no scenario where it's genuinely necessary.